fetch/client/src/test/resources/wiremock/mappings/mappings.json

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
I wonder if this file won’t be too difficult to maintain?

Reply:
I could think of ways to implement this to be more maintainable, for example by interacting with `WireMock` through HTTP API to submit JSON stubs from tests, but I am hesitant to commit more effort to this because it will be all made redundant rather quickly with the implementation of https://github.com/endpoints4s/endpoints4s/issues/945.

Reply:
OK, let’s keep it for now and see how it goes after #945 is solved.